consensus/istanbul/backend/api.go

```diff
@@ -42,6 +42,21 @@ type API struct {
 }
 
 // getHeaderByNumber retrieves the header requested block or current if unspecified.
+func (api *API) getHeaderByNumber(number *rpc.BlockNumber) (*types.Header, error) {
+	if number == nil || *number == rpc.LatestBlockNumber || *number == rpc.PendingBlockNumber {
```

Comment:
I guess you shouldn't really call this with pending.  Not sure if it's better to treat it the same as latest or to return an error.  Probably doesn't matter much either way.